INSTALL/Install_configure_postgreSQL_CENTOS7.md

```diff
@@ -0,0 +1,126 @@
+Following steps will let you install, configure postgreSQL server and create a
+`haas` database for development or production version of HaaS on a 
+Centos - 7 server
+
+
+
+## Part 1: 
+Install PostgreSQL server. Initialize the system. Configure PostgreSQL to allow password authentication.
+
+**1. Install the requisite packages on your server.**
+
+```
+sudo sudo yum install postgresql-server postgresql-contrib -y
+```
+
+**2. Initialize postgresql:**
+
+```
+sudo postgresql-setup initdb
+```
+
+**3. Replace the term `ident` from following lines**
+in file `/var/lib/pgsql/data/pg_hba.conf` with `md5`
+
+**Before:**
+
+```
+# "krb5", "ident", "peer", "pam", "ldap", "radius" or "cert".  Note that
+host    all             all             127.0.0.1/32            ident
+host    all             all             ::1/128                 ident
+#host    replication     postgres        127.0.0.1/32            ident
+#host    replication     postgres        ::1/128                 ident
+```
+
+**After:**
+
+```
+# "krb5", "ident", "peer", "pam", "ldap", "radius" or "cert".  Note that
+host    all             all             127.0.0.1/32            md5
+host    all             all             ::1/128                 md5
+#host    replication     postgres        127.0.0.1/32            ident
+#host    replication     postgres        ::1/128                 ident
+```
+
+**4. Start postgresql service**
+
+```
+$ sudo systemctl start postgresql
+$ sudo systemctl enable postgresql
+```
+
+##Part 2: 
+Setup system user and database role for HaaS.Create database `haas` owned by user `haas`.
+
+**5. Create a system user haas:**
```

Comment:
I agree that the procedure to create user `haas` needs to be in the corresponding installation procedure. But I feel it is important to repeat it here for the context. 

I know that system user and database role name can be different but having them same, simplifies management. A user does not have to remember which username for system or database separately. 
For eg. say after 6 months of setting up the system, the user needs to tell someone about managing haas. It will be simply as follows:
system user name managing haas is `haas`
Database username that owns and accesses postgreSQL is `haas`
Database name `haas`

This reduces configuration hassle. Also users are free to configure to do it any other way.
This is just one way to get started rather than getting confused on multiple ways available on the internet.